LFRIC/singularity/README.md

```diff
@@ -1 +1,89 @@
 # How to build a Singularity container of LFRIC
+
+This is built on previous work using Singularity to create a LFRic build system.
+LFRic has a large dependency tree and is currently being developed by anumber of users on a variety of systems, from laptops to HPCs.
+It is fully described at  https://github.com/NCAS-CMS/LFRic_container
+
+The aim of this project is to:
+
+1) Put the LFRic build environment into a portable package that can be deployed on different x86 target machines to build the LFRic executables. 
+
+2) Produce an executable that is able to make use of local MPI libraries and therefore local fast interconnects.
+
+3) Produce an executable that be run on the native system with no run-time dependencies on the build environment to minimise the conflict between the packaged libraries and local libraries used for MPI and job control.
+
+
+
+It is dependent on Intel Fortran and an MPICH-ABI compliant MPI implementation. 
+
+### Build container
+
+Build the Singularity LFRic build system as described in step 1-4 in https://github.com/NCAS-CMS/LFRic_container then copy to Piz Daint.
+
+### On Piz Daint:
+```
+unset LD_PRELOAD # There is a general LD_PRELOAD with interferes with Singularity, but isn't required for it.
+module swap PrgEnv-cray PrgEnv-intel
+module load singularity
+svn checkout --username <username> https://code.metoffice.gov.uk/svn/lfric/LFRic/trunk
+```
+
+For gungho the Makefile needs to be edited. Change the lines:
+```
+export EXTERNAL_DYNAMIC_LIBRARIES = yaxt yaxt_c netcdff netcdf hdf5 \
+                                    $(CXX_RUNTIME_LIBRARY)
+export EXTERNAL_STATIC_LIBRARIES = xios
+```
+to
+```
+export EXTERNAL_DYNAMIC_LIBRARIES = 
+export EXTERNAL_STATIC_LIBRARIES = yaxt yaxt_c xios netcdff netcdf hdf5_hl hdf5 
+ z :libstdc++.a
+```
+
+### Start the container
+```
+singularity shell -B /opt/intel:/opt/intel lfric_usr.sif #Start singularity with bind points for the local Intel compilers.
+```
+
+### The inside the container.
```

Comment:
"Inside of the container" or "Inside the container" (and no full-stop as this is a title)?